modules/serverless-install-web-console.adoc

```diff
@@ -18,7 +18,7 @@ image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web
 . On the *Install Operator* page:
 .. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
 .. The *Installed Namespace* will be `openshift-operators`.
-.. Select the *4.6* channel as the *Update Channel*. The *4.6* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
+.. Select the *4.7* channel as the *Update Channel*. The *4.7* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
```

Comment:
@warrenvw @alanfx will this be accurate for 4.7 do you know? I know sometimes our channels have been behind the release?

Reply:
@abrennan89 I'm reverting this change until we can get confirmation.